reactive/kubernetes_worker.py

```diff
@@ -966,7 +966,7 @@ def get_kube_api_servers():
 @when_any("kube-control.api_endpoints.available", "kube-api-endpoint.available")
 def update_nrpe_config():
     services = ["snap.{}.daemon".format(s) for s in worker_services]
-    data = render("nagios_plugin.py", context={"node_name": get_node_name()})
+    data = render("nagios_plugin.py", None, {"node_name": get_node_name()})
```

Comment:
using [this API](https://github.com/juju/charm-helpers/blob/master/charmhelpers/core/templating.py#L21)

instead of [this one](https://github.com/juju-solutions/charms.templating.jinja2/blob/master/charms/templating/jinja2.py#L27)